src/Funnelback/Response.php

```diff
@@ -221,4 +242,21 @@ protected function buildFacets($facets_data)
         }
         return $facets;
     }
+
+    /**
+     * Builds a list of facet metadata.
+     *
+     * @param array $facets_data
+     *   The raw facet data.
+     *
+     * @return array
+     *   A list of facet metadata
+     */
+    protected function buildFacetDefinitions($facets_data) {
```

Comment:
Curly brace should be dropped 👀

Reply:
Resolved via 75388949cb5632b39f0c9c8a9c7112879ab29b2e